src/Token.php

```diff
@@ -57,6 +57,7 @@ public static function getTokenKindNameFromValue($kindName) {
         foreach ($constants as $name => $val) {
             if ($val == $kindName) {
```

Comment:
we could consider doing it strictly, because then php doesn't need to convert it(because most of the time this IS false).